pages/Javascript-API.md

```diff
@@ -56,13 +89,69 @@ The properties of the feature at the current cursor location may be accessed thr
 
 This is accomplished by assigning a unique color to each feature onscreen and rendering the scene to an offscreen buffer. When queried, the `scene.selection` object checks the offscreen render at the current cursor position, and identifies the feature by its color.
```

Comment:
Additionally, this only works right now because of demo-specific code to add mouse cursor event handling, etc., this isn't in Tangram itself and won't work out of the box.